### PR TITLE
Throttle api requests

### PR DESCRIPTION
**Pull request by BenignBeppe**

### CODE REVIEW

File: hb_config.sample.py [hunk 1]
```diff
@@ -71,3 +71,6 @@
 # of looking for plain strings, these stings are used to flexibly
 # search for templates.
 skip_templates = ["blocked"]
+
+# Wait at least this long between sending invitations.
```
Comment (lokal-profil) [inline hb_config.sample.py:75]: I would add that it is in second and that it is between inviting each individual (as compared to between each run or between a second invitation to the same person)

File: scripts/teahouse/send_th_invites.py [hunk 2]
```diff
@@ -20,6 +20,8 @@
 
 import random
 import logging
+import imp
```
Comment (lokal-profil) [inline scripts/teahouse/send_th_invites.py:23]: Why do we need imp? isn't that only locally used in config_reader?

**Review by lokal-profil — CHANGES_REQUESTED** [review]
Looks good. Just 2 minor comments

**Comment by BenignBeppe** [comment]
The history for the branch is a mess, recreating.

**Pull request closed without merging**